tests/build-and-run-tests.sh

```diff
@@ -189,17 +200,33 @@ function detectChanges {
 function cloneCode {
     detectChanges
     git clone ../ "${BUILD_DIR}"
+}
+
+# Copy source code into build directory
+function copyCode {
+    rsync -aq ../ ${BUILD_DIR} --exclude='builds' --exclude='build-rkt-*'
+}
+
+# Set source code into build directory and enter into it
+function setCodeInBuildEnv {
+    if [[ ${DIRTYBUILD} == true ]]; then
+        copyCode
+    else
+        cloneCode
+    fi
```

Comment:
I was wondering why do we need the two options.
```
function setCodeInBuildEnv {
  copyCode
}
```

Do we need to keep `cloneCode`?